homework-submission/todo-app/src/component/week3Components/img.jsx

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+const Img = ({ imgSrc, imgAlt }) => (
+  <li className="img">
+    {" "}
```

Comment:
What's this space for?

Reply:
I don't know, when I saved the code, it was added by formatter.

Reply:
But, I deleted it.